doc/classes/PopupMenu.xml

```diff
@@ -91,6 +107,8 @@
 			<argument index="3" name="global" type="bool" default="false">
 			</argument>
 			<description>
+				Adds an icon and assigns the specified [ShortCut] and [Texture] to it.
+				If no [code]id[/code] is provided then an [code]id[/code] will be assigned to it automatically.
```

Comment:
Avoid passive voice and unclear antecedents (it -> the icon). See the [docs writing guidelines](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)

Reply:
I tried that but it seemed unorthodox? 
```
Adds a checkbox and assigns the specified [ShortCut] to it. The label of the checkbox will be the [ShortCut] name.
Checkbox does not have a default behavior. See [method set_item_checked] for more info on how to control the checkbox.
```
Too many "checkboxes". But i will try to reference the control name as much as possible (not always though)

Reply:
You can use the pronoun once here. It depends on the previous sentence. In the first sentence, you have two subjects: checkbox and shortcut. If you say `it` in the second sentence, it's not clear what you're referring to. But on the second line it works: it can only refer to `checkbox`.
 
`Adds a checkbox and assigns the specified [ShortCut] to it. Sets the checkbox's label to the [ShortCut]'s name.
The checkbox does not have a default behavior. See [method set_item_checked] for more info on how to control it.`